opentelemetry-kotlin-api/api/opentelemetry-kotlin-api.api

```diff
@@ -77,6 +77,9 @@ public abstract interface class io/embrace/opentelemetry/kotlin/creator/SpanCrea
 
 public abstract interface class io/embrace/opentelemetry/kotlin/creator/TraceFlagsCreator {
 	public abstract fun getDefault ()Lio/embrace/opentelemetry/kotlin/tracing/model/TraceFlags;
+	public abstract fun getRandom ()Lio/embrace/opentelemetry/kotlin/tracing/model/TraceFlags;
+	public abstract fun getSampled ()Lio/embrace/opentelemetry/kotlin/tracing/model/TraceFlags;
+	public abstract fun getSampledAndRandom ()Lio/embrace/opentelemetry/kotlin/tracing/model/TraceFlags;
```

Comment:
Same comments as for the `TraceFlags` implementation - I think a `create` function with two bool parameters would be a better fit here